src/Worker.php

```diff
@@ -99,14 +89,19 @@ private function tick($queue)
      * @param string $name
      * @param array $router
      *
-     * @return null|string
+     * @return null|callable
      */
     private function getHandler($name, array $router = [])
     {
         if (!isset($router[$name])) {
             return null;
         }
 
-        return $router[$name];
+        $handler = $router[$name];
+        if (!is_callable($handler)) {
+            return null;
```

Comment:
Seems like we should throw an exception here or something? This circumstance would generally indicate an error in configuration.